### ISSUE TITLE
Game 2 - payable keyword

### ISSUE THREAD

**Issue by Brivan-26** [ChainShot/Contract-Puzzles#4]
On [Contract 2](https://github.com/ChainShot/Contract-Puzzles/blob/master/contracts/Contract2.sol), the `switchOn` function has the `payable` keyword, but there's no ether transfer neither on the contract nor on the [test file](https://github.com/ChainShot/Contract-Puzzles/blob/master/test/game2Test.js).
I guess the *payable keyword* should be removed, or Am i missing something?

**Comment by sio-doh** [comment]
True. For functionality purposes **payable** keyword should be removed as no ether transfer is occurring.
